app/navigation/Navigator.scala

```diff
@@ -157,19 +159,43 @@ class Navigator @Inject() () {
         //
     case PrivateUseAdjustmentPage(Rentals) =>
       taxYear => _ => _ => BalancingChargeController.onPageLoad(taxYear, NormalMode, Rentals)
+    case PrivateUseAdjustmentPage(RentalsRentARoom) =>
+      taxYear => _ => _ => BalancingChargeController.onPageLoad(taxYear, NormalMode, RentalsRentARoom)
     case BalancingChargePage(Rentals) =>
       taxYear => _ => _ => PropertyIncomeAllowanceController.onPageLoad(taxYear, NormalMode, Rentals)
+    case BalancingChargePage(RentalsRentARoom) =>
```

Comment:
What about Rentals? can we make it parametrised?

Reply:
There are cases for both **Rentals** and **RentalsRentARoom** .  I am happy to parameterise it - personally I like to be a separate case to show the different property type